designer/client/model/data-model.js

```diff
@@ -0,0 +1,28 @@
+import { serializeAndDeserialize, clone } from '../helpers'
+
+export class Data {
```

Comment:
should we add data.save method to this class?

Reply:
To be honest I can't obviously see where the data.save method is currently defined, perhaps you can help me out with that? It does sound like a good candidate to move here, I imagine there are also others so perhaps we can move in future PR's?

Reply:
I have no idea why it's defined this way.. see index.js `App.save`-> `data.save = this.save`. As long as data.save is still being assigned on the new class instances we can leave for another PR. I'm not sure myself if assigning like this works, you'll have to check!

Reply:
Ok, will do!

Reply:
I wrote a test for it and it looks like the assigned function is not copied (though it already isn't in the old code, so everything still works). If we decide to move this later we'll need to consider any callbacks to the App state